app/main.py

```diff
@@ -2,0 +1,38 @@
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = 1) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = 0
```

Comment:
Set `self.is_hungry` to `False` instead of `0` to maintain consistency with boolean data types.